src/E2_economics.py

```diff
@@ -24,8 +24,12 @@
     COST_DISPATCH,
     COST_OM_FIX,
     COST_UPFRONT,
+    ENERGY_PRODUCTION,
+    TOTAL_FLOW,
 )
 
+LCOE = {}
```

Comment:
Is there a reason you create this variable as a global constant?

Reply:
When I didn't, it was telling me that LCOE is not defined

Reply:
also, I am not sure if I should use dict_asset or dict_values

Reply:
you should then define it within the function (and use small caps as convention for non-global variables)

Reply:
okay will do so now, thanks!

Reply:
> also, I am not sure if I should use dict_asset or dict_values

You named the function's argument `dict_asset` but you are passing  `dict_values` to the function when you use it. If you only need what is under `dict_values[ENERGY_PRODUCTION]` you can replace `dict_asset[ENERGY_PRODUCTION]` by `dict_asset` and pass `dict_values[ENERGY_PRODUCTION]` to your function (if I understand well `dict_asset` is `dict_values[ENERGY_PRODUCTION]` by definition)

Reply:
I think it's dict_values[ENERGY_PROVIDERS] from [here](https://github.com/rl-institut/mvs_eland/blob/f3b0e1562279baf4a66fb84f0b221ec71fcce058/src/C0_data_processing.py#L576) 
maybe I should just use dict_values[ENERGY_PRODUCTION] everywhere?

Reply:
See next comment from me below